server/src/test/java/org/apache/iotdb/db/integration/IoTDBMultiSeriesIT.java

```diff
@@ -334,4 +334,32 @@ public void crossSeriesReadUpdateTest() throws ClassNotFoundException, SQLExcept
       fail(e.getMessage());
     }
   }
+
+  @Test
+  public void selectUnknownTimeSeries() throws ClassNotFoundException {
+    Class.forName(Config.JDBC_DRIVER_NAME);
+
+    try (Connection connection = DriverManager
+        .getConnection(Config.IOTDB_URL_PREFIX + "127.0.0.1:6667/", "root", "root");
+        Statement statement = connection.createStatement()) {
+      statement.execute("select s10 from root.vehicle.d0");
+      fail("not throw exception when select unknown time series");
+    } catch (SQLException e) {
+      assertEquals("Execute statement error: Path: \"root.vehicle.d0.s10\" doesn't corresponding to any known time series", e.getMessage());
+    }
+  }
+
+  @Test
+  public void selectWhereUnknownTimeSeries() throws ClassNotFoundException {
+    Class.forName(Config.JDBC_DRIVER_NAME);
+
+    try (Connection connection = DriverManager
+        .getConnection(Config.IOTDB_URL_PREFIX + "127.0.0.1:6667/", "root", "root");
+        Statement statement = connection.createStatement()) {
+      statement.execute("select s1 from root.vehicle.d0 where s0 < 111 and s10 < 111");

```

Comment:
Better add a test for the case where filter paths start with ROOT. Correspond to the error exception you add in the class `MTree`.